packages/tabs/src/vaadin-tab.js

```diff
@@ -37,7 +37,18 @@ import { ThemableMixin } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mix
  */
 class Tab extends ElementMixin(ThemableMixin(ItemMixin(PolymerElement))) {
   static get template() {
-    return html`<slot></slot>`;
+    return html`
+      <style>
+        :host {
+          display: inline-flex;
```

Comment:
Should the default be `flex`?

![Screenshot 2022-06-03 at 11 35 44](https://user-images.githubusercontent.com/1222264/171819432-585fd8a2-b2ea-4f6f-b789-6fcdb82731c8.png)

Reply:
Shouldn't it be a simple `display: block`?

Reply:
I guess it's ok to use `display: block` as a default.